woocommerce/woocommerce-gateway-stripe
# feat: checks on notice compatibility

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1914

We were hoping to check the compatibility with the inbox notices by _just_ checking the WC version.
It can happen that some users disable WC admin via this code:
```
add_filter('woocommerce_admin_disabled', '__return_true');
```

So in that case, the WC version passes, but the notes class might not be available.
So, we should probably check for that instead.

# Testing instructions
- Add the filter mentioned above to your codebase
- The site should no longer crash with a fatal error

I was able to check the fatal error on `develop` by switching my WC version via the WooCommerce Beta Tester plugin: https://wordpress.org/plugins/woocommerce-beta-tester/
